os/drivers/lcd/lcd_dev.c

```diff
@@ -290,11 +293,32 @@ static int lcddev_ioctl(FAR struct file *filep, int cmd, unsigned long arg)
 	}
 	break;
 	case LCDDEVIO_SETPOWER: {
+
+#ifdef CONFIG_PM
+		if (!priv->dev->setpower || !priv->dev->getpower) {
+			ret = -ENOSYS;
+			break;
+		}
+		int new_power = (int)arg;
+		int old_power = priv->dev->getpower(priv->dev);
+
+		ret = priv->dev->setpower(priv->dev, new_power);
+		if (ret != OK) {
+			break;
+		}
+
+		if (old_power == 0 && new_power > 0) {
+			(void)pm_suspend(priv->pm_domain);
+		} else if (old_power > 0 && new_power == 0) {
+			(void)pm_resume(priv->pm_domain);
+		}
```

Comment:
When you tidy up the pm, let's use definition instead of 0.
like LCD_BACKLIGHT_OFF